interfaces/Portalicious/src/app/app.routes.ts

```diff
@@ -31,8 +31,8 @@ export const routes: Routes = [
     component: LoginComponent,
   },
   {
-    path: AppRoutes.allProjects,
-    component: AllProjectsComponent,
+    path: AppRoutes.projects,
+    component: ProjectsOverviewComponent,
```

Comment:
```suggestion
    path: AppRoutes.projects,
    title: $localize`:Browser-tab-title@@page-title-projects-overview:All Projects`,
    component: ProjectsOverviewComponent,
```

Oops, forgot that this could also be set already...